src/libs/Middleware/Logging.js

```diff
@@ -116,6 +117,10 @@ function Logging(response, request) {
                 // Expensify site is down completely OR
                 // Auth (database connection) is down / bedrock has timed out while making a request. We currently can't tell the difference between Auth down and bedrock timing out.
                 Log.hmmm('[Network] API request error: Expensify service interrupted or timed out', logParams);
+                throw error;
+            } else if (error.message === CONST.ERROR.THROTTLED) {
+                Log.hmmm('[Network] API request error: Expensify API throttled', logParams);
+                throw error;
```

Comment:
hmmmm, what will happen if we throw? Everything will break, no? Nothing is catching this

Reply:
Ah, it is caught in the retry middleware, right?